### PR TITLE
Tessera conf

### PR DESCRIPTION
**Pull request by macfarla**
Added 3 x tessera configs, using same keys, ports and urls as orion.

### CODE REVIEW

File: scripts/delete_databases.sh [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+echo 'Deleting Besu, Tessera and Orion databases'
 
 rm -rf workdir/orion1/routerdb workdir/orion1/orion.ports
 rm -rf workdir/orion2/routerdb workdir/orion2/orion.ports
 rm -rf workdir/orion3/routerdb workdir/orion3/orion.ports
 
+rm -rf workdir/orion1/tessera1*
+rm -rf workdir/orion2/tessera2*
+rm -rf workdir/orion3/tessera3*
+
```
Comment (lucassaldanha) [inline scripts/delete_databases.sh:11]: We should probably rename the directory to tessera :)

File: scripts/delete_databases.sh [hunk 2]
```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
-echo 'Deleting Besu and Orion databases'
+echo 'Deleting Besu, Tessera and Orion databases'
 
 rm -rf workdir/orion1/routerdb workdir/orion1/orion.ports
 rm -rf workdir/orion2/routerdb workdir/orion2/orion.ports
 rm -rf workdir/orion3/routerdb workdir/orion3/orion.ports
```
Comment (lucassaldanha) [inline scripts/delete_databases.sh:6]: Does Tessera create a ports file as well? We should delete the files created by Tessera.

Comment (macfarla) [reply scripts/delete_databases.sh:6]: good point. by default it doesn't but we are adding the ability to do this (will require a CLI option to be set)

File: scripts/delete_databases.sh (hunk 1 above)
Comment (macfarla) [reply scripts/delete_databases.sh]: updated

File: scripts/delete_databases.sh (hunk 2 above)
Comment (macfarla) [reply scripts/delete_databases.sh]: let's leave that change for another PR :)

**Pull request merged**